packages/@adobe/spectrum-css-temp/components/toggle/skin.css

```diff
@@ -224,3 +224,60 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-ToggleSwitch {
+    forced-color-adjust: none;
+    --spectrum-switch-emphasized-handle-background-color: ButtonFace;
+    --spectrum-switch-emphasized-handle-border-color: ButtonText;
+    --spectrum-switch-emphasized-handle-border-color-disabled: GrayText;
+    --spectrum-switch-emphasized-handle-border-color-down: Highlight;
+    --spectrum-switch-emphasized-handle-border-color-hover: Highlight;
+    --spectrum-switch-emphasized-handle-border-color-key-focus: ButtonText;
```

Comment:
Bit of a discrepancy here between the XD file and the implemented styles, specifically with the text color and the keyboard focus styles:
Storybook:
![switch](https://user-images.githubusercontent.com/9661127/175700140-fd0fc4f8-4933-48bd-87d7-550093284f7b.PNG)


XD:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/9661127/175700103-8cc9741e-8395-4b1d-a6c3-9338f3bc8550.png">

IMO, I think the current text style seem more appropriate, kinda weird that the XD file has text as the selected text color. However, what about the keyboard focus style when the switch isn't selected? Should it switch to highlight like hover?

Reply:
Lemme ask Spectrum Design to double check

Reply:
Update: Spectrum will get back to us on this specific case

Reply:
Spectrum has confirmed that the current behavior in this storybook are correct, the XD file had the incorrect colors